spec/test_utils.rb

```diff
@@ -0,0 +1,106 @@
+module TestUtils
+  def create_session_with_streams_and_measurements!(attributes = {})
```

Comment:
Could you please check if you can avoid passing some attributes to the create_* methods? For example `id` in session I guess can be skipped

Reply:
Yes, it's used in many tests, and one of the passes session id.
But in general, there are some more tests that can be refactored now to use this file, but it requires some adjustments, so I didn't want to do all of them at once.